src/lib/tenant/tenantUIFactory.ts

```diff
@@ -67,12 +77,20 @@ const etherfiUI = new TenantUI({
   toggles: [
     {
       name: "delegates",
-      enabled: false,
+      enabled: true,
+    },
+    {
+      name: "info",
+      enabled: true,
     },
     {
       name: "proposals",
       enabled: false,
     },
+    {
+      name: "delegates/edit",
```

Comment:
this is interesting pattern and maybe we should talk about this -- but my gut tells me if I were to think about this feature I'd probably have done something like updating the delegates toggle to 

```
{
      name: "delegates",
      enabled: false,
      config: {
        edit: false
      }
}
```

to me, each of these feels like a feature, and being able to edit delegates doesn't feel so much as an independent feature as it is an action of the delegation feature.

this becomes way more complex with my proposal lifecycle generalization work where there is much more you can customize from tenant to tenant, and I think the `config` key here makes sense.

Not necessary to change but flagging it as something I'd like to talk to you about for the PLM work sometime.